modules/load-balancer-target-group/main.tf

```
@@ -54,10 +54,38 @@ resource "aws_alb_listener_rule" "http_path" {
   }
 
   dynamic "condition" {
-    for_each = var.routing_condition
+    for_each = [for condition in var.routing_condition : condition.values if condition.field == "path_pattern"]
```

We probably want to keep these as dashes for backwards compatibility.

```suggestion
    for_each = [for condition in var.routing_condition : condition.values if condition.field == "path-pattern"]
```

---

Done: https://github.com/gruntwork-io/terraform-aws-couchbase/pull/64/commits/c6e8fd92f23ff3dd19af47c461d835860b9df32c